pyproject.toml

```diff
@@ -5,3 +5,9 @@ requires = [
 ]
 
 build-backend = "setuptools.build_meta"
+
+
+[tool.pytest.ini_options]
+markers = [
+    "allow_empty(platform): Allow main tests on this platform(s) to be empty",
```

Comment:
Still not a fan of the marker, actually. Especially because we don't want to document it here. It's an internal thing, and not part of the interface, so nothing that should need to be added here.